scripts/process_notebooks.py

```diff
@@ -362,9 +340,10 @@ def clean_notebook(nb):
                 cell["metadata"].pop("id")
 
         if cell["cell_type"] == "code":
-
-            # Remove code cell outputs
-            cell["outputs"] = []
+            
+            if flag:
+                # Remove code cell outputs only in tutorials
```

Comment:
Better to separate the logic here ... this function doesn't need to "know" that output clearing relates to tutorials ... it should just clear outputs if requested, and otherwise not.

Reply:
:+1: :100: